src/components/CommentForm/CommentForm.tsx

```diff
@@ -475,6 +477,24 @@ export const CommentForm = ({
                             >
                                 i
                             </button>
+                            <button
```

Comment:
Could you add the Ophan attributes for this? Think the examples are in master as can't see them in the buttons above

Reply:
If I look on frontend these button are li tags with no ophan stuff on them

```
<li class="d-comment-box__formatting-bold" title="Bold">B</li>
```

Clicking does send an ophan event but it seems to be a generic one, with no extra params, that gets sent on every click event, anywhere within discussion

```
https://ophan.theguardian.com/img/2?viewId=k9887d5ufcybupkecl0d&clickComponent=discussion
```

Reply:
Yup, it's new tracking, see https://github.com/guardian/discussion-rendering/pull/221/files#diff-224102f30975c2ad7f39e9be6c1c8d76R472

Reply:
Oh! discussion `master`! Gotcha

Reply:
Done.

Nice, now I get to see how popular my new formatting options are!